README.md

```diff
@@ -18,6 +18,8 @@ The HERO software development kit (SDK) contains the following submodules:
 The HERO SDK currently supports the [Xilinx Zynq ZC706 Evaluation Kit](https://www.xilinx.com/products/boards-and-kits/ek-z7-zc706-g.html) as target hardware platform.
 
 ### Software
+First, make sure your shell environment does not deviate from the system defaults regarding paths (`PATH`, `LD_LIBRARY_PATH`, etc) before setting up the SDK or sourcing the environment of the HERO SDK.
```

Comment:
if I have to be honest, I think the disclaimer could be misleading. The root cause of the problem issued #40 is the fact that `cmake` version used was not compatible with our build flow. Then secondarily the problem was caused by the Vivado's environment that could change - in a stealth mode - the program versions compare the default ones.

Reply:
I see #40 as one specific instance of a more general workflow problem. A similar problem could have faced when a user sources the Vivado environment after sourcing the HERO environment, and the ARM compiler in the Vivado SDK is invoked instead of HERO's. So the very basic rule is: Non-default paths should be avoided.